web_dashboard_tile/__manifest__.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# © 2010-2013 OpenERP s.a. (<http://openerp.com>).
+# © 2014 initOS GmbH & Co. KG (<http://www.initos.com>).
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    "name": "Dashboard Tile",
+    "summary": "Add Tiles to Dashboard",
+    "version": "11.0.1.0.0",
+    "depends": [
+        'web',
+        'board',
+        'mail',
+        'web_widget_color',
+    ],
+    'author': 'initOS GmbH & Co. KG, '
+              'GRAP, '
+              'Odoo Community Association (OCA)',
+    "category": "web",
+    'license': 'AGPL-3',
+    'contributors': [
```

Comment:
I don't think this works. You can use 'maintainers' though https://github.com/OCA/maintainer-tools/blob/771508e99876115c418888bbac7899174662482b/template/module/__manifest__.py#L14